R/Task_operators.R

```diff
@@ -44,7 +43,7 @@ getTaskType = function(x) {
 #' @export
 #' @family task
 getTaskId = function(x) {
-  getTaskDesc(x)$id
+  getTaskDescription(x)$id
```

Comment:
This  happend by the automatic merging. Make sure that the only code changes in the Diff is stuff relevant for your PR